schema.json

```diff
@@ -632,7 +632,8 @@
         },
         "lastModified": {
           "type": "string",
-          "description": "Using ISO 8601 with YYYY-MM-DDThh:mm:ss"
+          "description": "Using ISO 8601 with YYYY-MM-DDThh:mm:ss",
```

Comment:
The description could use the example + description format we have in the other fields. Also we probably should document the format for these description strings like a spec.